apis/python/src/tiledbsoma/sparse_nd_array.py

```diff
@@ -124,6 +124,7 @@ def create(
             ctx=self._ctx,
         )
 
+        # TODO: confirm that no timestamp is needed or supported for tiledb.Array.create()
         tiledb.Array.create(self._uri, sch)
```

Comment:
timestamps appear not yet supported for (i) Array creation and (ii) any group operation -- please correct me if I've got that wrong? @thetorpedodog @Shelnutt2